tests/adapters/test_visa.py

```diff
@@ -37,8 +37,10 @@
     pytest.skip('PyVISA tests require the pyvisa-sim library', allow_module_level=True)
 
 
-def make_visa_adapter(**kwargs):
-    return VISAAdapter(SIM_RESOURCE, visa_library='@sim', **kwargs)
+@pytest.fixture
+def adapter():
+    return VISAAdapter(SIM_RESOURCE, visa_library='@sim',
+                       read_termination="\n")
```

Comment:
The read_termination should be set by the individual SIM_RESOURCE -- 'ASRL2::INSTR is built-in in pyvisa-sim, I think

Reply:
the SIM_RESOURCE just defines how the virtual device behaves. The default device (ASRL2) writes "\n". That does not define, how pyvisa (the computer end) communicates with the virtual device. Therefore, I have to tell the VISAAdapter what to expect.